CreeDictionary/DatabaseManager/__main__.py

```diff
@@ -46,33 +46,56 @@ def add_multi_processing_argument(ap: ArgumentParser):
     "xml_directory_name", help="The directory that has crkeng.xml and engcrk.xml"
 )
 
+for p in [build_test_db_parser, import_parser]:
+    p.add_argument(
+        "--noinput",
+        "--no-input",
+        dest="no_input",
+        action="store_true",
+        help="Do NOT interactively prompt the user anything",
+    )
 add_multi_processing_argument(import_parser)
 
 
-def import_and_migrate(xml_dir_name: Union[Path, str], process_count: int):
+def import_and_migrate(
+    xml_dir_name: Union[Path, str], process_count: int, no_input=False
+):
     """
     Unlink existing .sqlite3 file, apply migrations from API/0001 to API/0005
-    Import from specified engcrk.xml and crkeng.xml. Migrate the rest from all apps
+    Import from specified engcrk.xml and crkeng.xml. Then migrate all other migrations
     """
+    if not no_input:
+        user_in = input(
+            f'this will DELETE database {settings.DATABASES["default"]["NAME"]}\n '
+            "and create a new database from scratch.\n"
+            "Are you sure? [y/N] "
+        )
```

Comment:
This part is interactive thus uncovered. (I can write tests with stdin and test it interactively but it's too much effort. That's why the coverage stonks.